docs/binary_input.md

```diff
@@ -34,28 +34,6 @@ and rescale the data.
 
```

Comment:
`Support for binary inputs and outputs data in OpenVINO Model Server` title is misleading, remove `outputs` word